uimaj-core/pom.xml

```diff
@@ -204,8 +204,13 @@
       <artifactId>asm-tree</artifactId>
       <version>5.0.4</version>
     </dependency>
- -->		
-
+ -->
```

Comment:
This sounds like a general new kind of support for testing; perhaps it should go higher up in the factoring, e.g., in the uimaj-parent pom?

Reply:
I added the dependency management to fix the version of this dependency to the parent pom, but I tend to keep dependencies where they are needed - even test dependencies.

Reply:
ok